pkg/commands/loaders/remotes.go

```diff
@@ -31,7 +31,12 @@ func NewRemoteLoader(
 }
 
 func (self *RemoteLoader) GetRemotes() ([]*models.Remote, error) {
-	remoteBranchesStr, err := self.cmd.New("git branch -r").DontLog().RunWithOutput()
+	remoteBranchesCommand := "git branch -r"
+	if self.Common.UserConfig.Git.SortRemoteBranchesByLastCommit {
+		remoteBranchesCommand = "git for-each-ref --sort=-authordate refs/remotes --format=\"%(refname:lstrip=2)\""
```

Comment:
How does this compare with `git branch -r` in terms of running time? If it's basically the same, I'd use `git for-each-ref` in both cases (passing whatever sort arg you need to get the behaviour you want)